chainstate/src/detail/transaction_verifier/mod.rs

```diff
@@ -533,7 +533,7 @@ impl<'a, S: BlockchainStorageRead> TransactionVerifier<'a, S> {
                 self.precache_inputs(tx.inputs())?;
 
                 // check for attempted money printing
-                self.check_transferred_amounts_and_get_fee(tx)?;
+                let _ = self.check_transferred_amounts_and_get_fee(tx)?;
```

Comment:
Should check return () instead of Amount that is ignored anyway?

Reply:
This is being used in two different contexts (IIRC), one of which requires a return value and the other just does the check.

Reply:
oh it returns fees, then ignore this